pkg/ring/model_test.go

```diff
@@ -104,23 +104,23 @@ func TestClaimTokensFromNormalizedToNormalized(t *testing.T) {
 	r := normalizedSource()
 	result := r.ClaimTokens("first", "second", true)
 
-	assert.Equal(t, []uint32{100, 200, 300}, result)
+	assert.Equal(t, []uint32{100, 200, 300}, []uint32(result))
```

Comment:
Why don't we use `Tokens` data type, like:

```
assert.Equal(t, Tokens{100, 200, 300}, result)
```

_Same comment applies to the rest of the file._